pkg/kapp/diff/change_set_with_versioned_rs.go

```diff
@@ -35,14 +39,20 @@ func (d ChangeSetWithVersionedRs) Calculate() ([]Change, error) {
 	existingRs := existingVersionedResources(d.existingRs)
 	existingRsGrouped := d.groupResources(existingRs.Versioned)
 
+	existingNonVerRsGrouped := d.groupNonVerResources(existingRs.NonVersioned)
```

Comment:
We are not grouping resources here, we are just creating a map of resource id and resource, maybe we should just do it inline in the `checkForMaxDuration` function directly?